front/app/api/community_monitor/useCommunityMonitorProject.ts

```diff
@@ -1,22 +1,28 @@
 import { useQuery } from '@tanstack/react-query';
 import { CLErrors } from 'typings';
 
-import projectsKeys from 'api/projects/keys';
 import { IProject, ProjectsKeys } from 'api/projects/types';
 
 import fetcher from 'utils/cl-react-query/fetcher';
 
+import communityMonitorKeys from './keys';
+
 const fetchCommunityMonitorProject = () => {
   return fetcher<IProject>({
     path: '/projects/community_monitor',
     action: 'get',
   });
 };
 
-const useCommunityMonitorProject = (id?: string) => {
+type Props = {
+  enabled?: boolean;
+};
+
+const useCommunityMonitorProject = ({ enabled }: Props) => {
   return useQuery<IProject, CLErrors, IProject, ProjectsKeys>({
-    queryKey: projectsKeys.item({ id }),
+    queryKey: communityMonitorKeys.all(),
     queryFn: () => fetchCommunityMonitorProject(),
```

Comment:
Minor: Just seeing this but since there are no parameters to pass

```suggestion
    queryFn: fetchCommunityMonitorProject,
```